Content.Shared/_RMC14/Projectiles/RMCProjectileSystem.cs

```diff
@@ -204,6 +200,11 @@ ent.Comp.Origin is { } origin &&
             delta.Length() > 0)
         {
             coordinates = coordinates.Offset(delta.Normalized() / -2);
+
+            if (HasComp<RMCFireProjectileComponent>(ent))
+            {
+                coordinates = coordinates.Offset(delta.Normalized()); // Apparently that works...
+            }
```

Comment:
This is terrible but it works for now, would have to check on different tickrates to see on which it is an issue